packages/qgrid-ngx/src/lib/vscroll/vscroll-column.directive.ts

```diff
@@ -36,7 +36,6 @@ export class VscrollColumnDirective implements OnDestroy, OnChanges {
   ngOnChanges(changes: SimpleChanges) {
     if (changes['index']) {
       if (this.port.getItemSize()) {
-        this.ngOnChanges = null;
```

Comment:
i'm afraid this would break something.
If you can't assign this method to null create a flag variable and after first visit set it ti true;
```typescript
private onChangesRan = false;
...
ngOnChanges(changes: SimpleChanges) {
  if (this.onChangesRan) {
    return;
  }
  if (changes['index']) {
    if (this.port.getItemSize()) {
      this.onChangesRan = true;
    }
  }
}
```